scripts/helpers/utils.ts

```diff
@@ -146,8 +157,10 @@ export async function getMerkleTreeUtils() {
     return MerkleTreeUtils.at(merkleTreeUtilsAddr);
 }
 
-export async function getRollupUtils() {
-    var rollupUtils: any = await rollupUtils.deployed();
+export async function getRollupUtils(rollupUtilsInstance?: any) {
+    var rollupUtils: any = !rollupUtilsInstance
+        ? await rollupUtils.deployed()
+        : rollupUtilsInstance;
```

Comment:
Can we wrap these checks in a function?

Reply:
Might have to create 3-4 functions for different contracts.